epages/auth.py

```diff
@@ -8,14 +8,22 @@
 
 
 def calculate_signature(code, access_token_url, client_secret):
-    message = '%s:%s' % (code, access_token_url)
-    digest = hmac.new(client_secret, msg=message, digestmod=hashlib.sha256).digest()
-    return base64.b64encode(digest).decode()
+    message = b'%s:%s' % (code, access_token_url)
```

Comment:
I'd rather pass (unicode) strings as parameters and encode to bytestrings in the function to have an easier-to-use interface